src/models/mockedTrains.go

```diff
@@ -2,10 +2,11 @@ package models
 
 import (
 	"github.com/satori/go.uuid"
+	"net/url"
 )
 
 func MockedGetTrains() {
```

Comment:
Could you add arguments such as trains and err to MockedGetTrains() and return them?
It will help to write unit tests in different cases.